### DISCUSSION TITLE
Open source contribution

### DISCUSSION THREAD

**Discussion by umairjavaid** [google/orbit#4448] [Q&A]
I came across this repo while I was working looking for a profiling tool. I want to study profilers in depth. I'm willing to make any opensource contribution to this project if someone could guide me or give me a module to work on that would be great!

**Comment by florian-kuebler** [comment]
Hey, contributions are always more than welcome. Unfortunately, we don't have an "easy to start with" issue list.

We suggest you could start by checking out the project, build it, and actually use the profiler on Linux in depth. If you stumble across something when using Orbit, raise an issue and we can discuss if this is something you could contribute to.
